conans/client/importer.py

```diff
@@ -56,6 +56,9 @@ def _report_save_manifest(copied_files, output, dest_folder, manifest_name):
         date = timestamp_now()
         file_dict = {}
         for f in copied_files:
+            if os.path.islink(f) and not os.path.exists(os.readlink(f)):
```

Comment:
from my experience, it might not be safe as `os.readlink` may raise (especially on Mac).
also, why fail hard in case of a broken link here? maybe a warning will be better?

Reply:
> from my experience, it might not be safe as os.readlink may raise (especially on Mac).

Correct, `readlink` is not safe, and that's why `os.path.islink` is verified before.

> why fail hard in case of a broken link here? maybe a warning will be better?

A broken link will raise a FileNotFound exception with stack trace which doesn't help user. This PR tries to show a better message for that error.